linux_os/guide/system/software/integrity/crypto/var_ssh_client_rekey_limit_time.var

```diff
@@ -0,0 +1,14 @@
+documentation_complete: true
+
+title: 'SSH client RekeyLimit - size'
+
+description: 'Specify the size component of the rekey limit.'
```

Comment:
Should expand on this a little more by adding (string) and mentioning the sizing style for minutes, hours, etc.